userscript/src/mods/autoMute.ts

```diff
@@ -120,20 +115,14 @@ export class AutoMuteMod extends Mod<{
     }
   }
 
-  private onPlayerJoined(e: CustomEvent): void {
-    if (Ninja.isGuest()) {
-      return;
-    }
+  private onPlayerJoined(e): void {
```

Comment:
I'll type these as "any" for now, but they should have their own typing in the future.